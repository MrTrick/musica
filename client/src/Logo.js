import React from 'react';
import logo from './logo.svg';

function Logo(props) {
  return (<img src={logo} aria-label="Musica Logo" alt="Musica Logo" />);
}

export default Logo;
