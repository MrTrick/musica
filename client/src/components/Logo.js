import React from 'react';
import logo from '../assets/logo.svg';

function Logo(props) {
  return (<img src={logo} aria-label="Musica Logo" alt="Musica Logo" />);
}

export default Logo;
