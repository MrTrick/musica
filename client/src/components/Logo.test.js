import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Logo from "./Logo";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Logo />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("looks like a logo should", () => {
  const component = renderer.create(<Logo />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
