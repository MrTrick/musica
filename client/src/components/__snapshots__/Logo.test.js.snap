// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`looks like a logo should 1`] = `
<img
  alt="Musica Logo"
  aria-label="Musica Logo"
  src="logo.svg"
/>
`;
